src/middleware/schema.js

```diff
@@ -66,7 +66,7 @@ const validateSchemas = ({validator, absoluteSchemas, rootUrl, serviceName, entr
     res.reply = (json) => {
       if (!req.hasAuthed) {
         const err = new Error('Deferred auth was never checked!');
-        return res.reportInternalError(err);
+        return next(err);
```

Comment:
I think this is another case of calling `next` for one handler when another handler is in progress -- `res.reply` gets called later, after the handler defined on line 44 is complete.  In #149 I used `throw err` here.  That thrown exception will be caught by `handle.js` and passed on to the error handler.